### PR TITLE
UPSTREAM: <carry>: openshift: create git history verification script

### PR DESCRIPTION
**Pull request by ingvagabund**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by enxebre** [comment]
/approve
we should probably consider to  move this to cluster-api-actuator-pkg

File: hack/verify_history.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Verifies git commits starting from predefined prefix.
+# Script is meant to be run in CI. Locally it can produce confusing outputs when local "master" branch is not in sync with remote one
+
+PREFIX="UPSTREAM: <[[:alnum:]]+>: openshift:"
```
Comment (frobware) [inline hack/verify_history.sh]: /lgtm

Do we want `<123>` supported? https://jira.coreos.com/browse/CLOUD-622

Comment (ingvagabund) [reply hack/verify_history.sh]: It's already enforced in openshift/cluster-api. I would be rather for relaxing the constraint to allow both `<123>` and `123`. Though, we can do it later.

**Pull request merged**